packages/opentelemetry-exporter-jaeger/src/jaeger.ts

```diff
@@ -82,6 +82,10 @@ export class JaegerExporter implements SpanExporter {
     return this._shutdownOnce.call();
   }
 
+  forceFlush(): Promise<void> {
+    return Promise.resolve();
```

Comment:
Same as OTLP if it is possible then ForceFlush should only resolve when any in-flight exports are complete.